packages/input-tel-dropdown/src/LionInputTelDropdown.js

```diff
@@ -301,9 +301,6 @@ export class LionInputTelDropdown extends LionInputTel {
       setTimeout(() => {
         this._inputNode.focus();
       });
-    } else {
```

Comment:
hmm, unfortunately this will give weird behavior in browser

Reply:
Maybe we should explicitly test this actually, so that it's captured in our tests as well

Reply:
Yes, was already thinking about it